### DISCUSSION TITLE
How to setup scrub to scrub full array

### DISCUSSION THREAD

**Discussion by vmazmaz** [auanasgheps/snapraid-aio-script#136] [Q&A] [answered]
I think my question is more snapraid-related than AIO-related, so feel free to remove. Basically, I run a scrub with every sync but I cannot get the last % of my array to get scrubbed, I sit at 17/18% of the array not scrubbed and every scrub does not bring this number down. Any suggestion ? Thanks

My configuration is as follows:
SCRUB_PERCENT=6
SCRUB_AGE=20
SCRUB_NEW=0
SCRUB_DELAYED_RUN=0

**Comment by tehniemer** [comment, marked as answer]
Set `SCRUB_NEW` to 1.

With your current settings it's not scrubbing any data newer than 20 days old, which is probably the 17/18% of your array.

**Reply by vmazmaz** [reply]
Thanks for your answer, I was not sure if the SCRUB_NEW parameter was the correct option.
